torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -536,6 +561,27 @@ void LoopNestGenerator::generate(const std::vector<Expr*>& exprs) {
   }
 }
 
+void LoopNestGenerator::cleanSharedMemory() {
+  for (auto& item : smem_) {
+    item.second = false;
+  }
+}
+
+void LoopNestGenerator::modifySharedMemory(Val* const key) {
+  auto it = smem_.find(key);
+  if (it != smem_.end()) {
+    it->second = true;
+  }
+}
+
+bool LoopNestGenerator::statusSharedMemory(Val* const key) const {
```

Comment:
same (remove the const in the `key` definition)

Reply:
done